### DISCUSSION TITLE
Show https-website that uses a self-created certificate

### DISCUSSION THREAD

**Discussion by Diavolezza** [sindresorhus/Plash#120] [Q&A] [answered]
I'm running a self hosted website using https with a self-created certificate. Plash shows an error, that it is not able to show this page due to security reasons. Using Safari, I also get an error but I can proceed and display the website anyway. Is there a possibility to show that page using Plash? Thank you!

**Comment by sindresorhus** [comment, marked as answer]
There is a setting to allow self-signed certificate in Plash 2.13.1.
